aikau/src/main/resources/alfresco/layout/AlfTabContainer.js

```diff
@@ -295,80 +295,99 @@ define(["dojo/_base/declare",
       delayProcessingDefault: true,
 
       /**
+       * Creates the "dijit/layout/TabContainer" wrapped by this widget and sets up associated
+       * subscriptions, etc.
+       * 
        * @instance
+       * @since 1.0.46
        */
-      postCreate: function alfresco_layout_AlfTabContainer__postCreate() {
+      createTabContainer: function alfresco_layout_AlfTabContainer__createTabContainer() {
          // Initialise a TabContainer instance and watch its selectedChildWidget event
-         this.tabContainerWidget = new TabContainer({
-            id: this.id + "_TABCONTAINER",
-            style: "height: " + this.height + "; width: " + this.width + ";",
-            doLayout: this.doLayout
-         }, this.tabNode);
-         this.tabContainerWidget.watch("selectedChildWidget", lang.hitch(this, this._tabChanged));
-
-         // Setup child widgets and startup()
-         if (this.widgets)
+         try
          {
-            // By default we want to ensure that we don't unnecessarily process widgets for 
-            // tabs that are not immediately visible. Therefore unless specifically requested 
-            // in the configuration to be the selected tab or the to render immediately then
-            // we'll mark them all as delayed processing. If no tab is marked as selected then
-            // we'll ensure that the first tab is both selected and will be immediately rendered
-            var tabSelected = false;
-            array.forEach(this.widgets, function(widget) {
-               if ((widget.delayProcessing === true && !widget.selected) || 
-                   widget.delayProcessing === false)
-               {
-                  // No action, use the explicit configuration
-               }
-               else if (widget.delayProcessing === true && widget.selected)
-               {
-                  // Silly configuration, if the tab is to be initially selected, processing can't be delayed...
-                  widget.delayProcessingDefault = false;
-               }
-               else
+            this.tabContainerWidget = new TabContainer({
```

Comment:
`this.create[...]` for the pubSub inheritance? (assuming already considered, but just flagging in case)